backend/corpora/common/utils/cloudfront.py

```diff
@@ -5,30 +5,35 @@
 
 import logging
 
-client = boto3.client('cloudfront')
+client = boto3.client("cloudfront")
+
 
 # Since Cloudfront is only used in deployed environments (dev, staging, prod),
 # only trigger an invalidation if the distribution_id is defined in secrets manager.
 # Otherwise this will be a no-op
 def create_invalidation(paths: list[str]):
-    distribution = CorporaCloudfrontConfig().distribution_id
-    if distribution:
+    try:
+        distribution = CorporaCloudfrontConfig().distribution_id
         return _create_invalidation(distribution, paths)
-    
+    except Exception:
```

Comment:
It would be an attribute error if distribution_id is not found. Also, you should move `_create_invalidation(distribution, paths)` out of the try block based on the warning you are giving.

Reply:
I don't get what you mean. This catches the case where `CorporaCloudfrontConfig().distribution_id` is not defined (which will only happen in rdev), and yield a no-op.